static/js/publisher/release/components/releasesConfirmDetails/progressiveRow.js

```diff
@@ -54,15 +65,21 @@ const ProgressiveRow = ({
   } else {
     progress = (
       <Fragment>
-        <InteractiveProgressiveBar
-          percentage={startingPercentage}
-          onChange={updatePercentage}
-          targetPercentage={targetPercentage}
-          minPercentage={1}
-          singleDirection={type === progressiveTypes.UPDATE ? 1 : 0}
-        />
+        {globalPercentage &&
+          !updateGlobalPercentage && (
+            <ProgressiveBar percentage={globalPercentage} disabled={true} />
```

Comment:
I realise it’s not a new problem with this change, but enabled and disabled sliders look very similar. Would be good if disabled sliders could be greyed out.

Reply:
Added a new issue here: https://github.com/canonical-web-and-design/snapcraft.io/issues/2398